technical_docs/Claims.md

```diff
@@ -71,26 +141,37 @@ This is equivalent to the following URI-form:
 
 ```
 {
-   "scope":                         "https://scitokens.org/v1/scope/read",
-   "https://scitokens.org/v1/path": "/",
-   "iss":                           "https://ligo.org/oauth"
+   "https://scitokens.org/v1/authz": "https://scitokens.org/v1/authz/read",
+   "https://scitokens.org/v1/path":  "/",
+   "iss":                            "https://ligo.org/oauth"
 }
 ```
 
-Note that the `path` claim is implicitly relative to a base authorization for the LIGO organization.  Here, `/` allows access to all LIGO files, _not_ all files in the storage service..
+Note that the `path` claim is implicitly relative to a base authorization for the LIGO organization.  Here, `/` allows access to all LIGO files, _not_ all files in the storage service.
 
 To stageout to `/store/user/bbockelm`, a part of the CMS namespace at the CMS site `T2_US_Nebraska`, a user would utilize the following token:
 
 ```
 {
-   "scope": "write",
+   "authz": "write",
    "path":  "/store/user/bbockelm",
    "iss":   "https://cms.cern/oauth",
    "site":  "T2_US_Nebraska"
 }
 ```
 
-To further restrict the permission given to an individual job, one may chose to further restrict the above token to a specific sub-directory via chaining tokens:
+The implementation of `site` is purposely ambiguous; hence, the `aud` claim may be used to restrict a token to a specific endpoint:
```

Comment:
I'm confused as to why we need both site and aud. The aud claim is a StringOrURI value. It could contain T2_US_Nebraska if that is the use case.